### PR TITLE
Dependency cleanup

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by Alexey-Kamenev — APPROVED** [review]
LGTM!

**Review by ktangsali — APPROVED** [review]
Looks good. Let's add a comment to CHANGELOG for this (since this is touching dependencies)

File: pyproject.toml
Comment (ktangsali) [inline pyproject.toml:1]: Also, wasn't raumel also added due to SFNO? Do we still need it?

Comment (mnabian) [reply pyproject.toml:1]: Updated the changelog. `raumel` is still a dependency of CorrDiff, so let's keep it for now. But in a couple of weeks I'll remove that as well.

Comment (ktangsali) [reply pyproject.toml:1]: Okay sounds good. Thanks.

**Pull request merged**